app/src/main/java/com/chartiq/chartiqsample/MainActivity.java

```diff
@@ -65,7 +65,7 @@
     private static final int STUDIES_REQUEST_CODE = 2;
     private static final int CHART_OPTIONS_REQUEST_CODE = 3;
     private static final String defaultSymbol = "AAPL";
-    public static final String chartUrl ="http://yourdeployment/sample-template-native-sdk.html";
+    public static final String chartUrl = "http://yourdeployment/sample-template-native-sdk.html";
```

Comment:
Is it ok to ask the developer to find these urls deep in the code and change them?  Would it be better to import this from a config file?

Reply:
Ideally yes. Since this is client side demo code it's encouraged that they change to how they see fit. I can put in an "enhancement" ticket to make a config file to place the url and other constants. Not worth changing it in this PR since all the projects would need changing.